cloud_experts_tutorials/cloud-experts-rosa-hcp-activation-and-account-linking-tutorial.adoc

```diff
@@ -0,0 +1,201 @@
++
+image::rosa-terms-conditions-9-5.png[]
++
+Submit your agreement once you have reviewed any additional terms when prompted at this time.
+
+. The {hybrid-console-second} provides a confirmation that AWS prerequisites were completed and lists the first steps needed for cluster deployment:
++
+image::rosa-cluster-create-10.png[]
+
+. The following steps pertain to technical deployment of the cluster:
++
+image::rosa-deploy-11.png[]
++
+* It is possible that these steps will be performed on a different machine than where the service enablement and account linking were completed.
+* As mentioned previously, any Red Hat account belonging to the Red Hat organization that was linked with the AWS account that activated the ROSA service will have access to creating a cluster and will be able to select the billing AWS account that was linked under this Red Hat organization previously.
++
+The last section of this page shows cluster deployment options, either using the `rosa` CLI or through the web interface:
++
+image::rosa-cli-ui-12.png[]
++
+* The following steps describe cluster deployment using the `rosa` CLI.
+* If you are interested in deployment using the web interface only, you can skip to the _{hcp-title} cluster deployment using the web interface_ section. However, note that the `rosa` CLI is required for certain tasks, such as creating the account roles. If you are deploying ROSA for the first time, follow this the CLI steps until running the `rosa whoami` command, before skipping to the web interface deployment steps.
+
+== {hcp-title} cluster deployment using the CLI
+
+. Click the *Download the ROSA CLI* button to download the ROSA command line interface (CLI) for your operating system and set it up as described in the xref:../cli_reference/rosa_cli/rosa-get-started-cli.adoc#rosa-get-started-cli[Help with ROSA CLI setup].
++
+[IMPORTANT]
+====
+Make sure that you have the most recent AWS CLI installed. See link:https://aws.amazon.com/cli/[Instructions to install the AWS CLI] for more information.
+====
++
+. After the previous steps are completed, you can verify that both CLI are available by running the `rosa version`. This command shows an update notification if you are using an older version and `aws –version` commands in your terminal.
+
+. The prerequisite for creating a {hcp-title} cluster is to log in using the `rosa` cli by the personalized command with your unique token shown under _2.1. To authenticate, run this command_. Use the *copy* button for easy copy and pasting of the command with full token into your terminal:
++
+image::rosa-token-13.png[]
++
+Do not share your unique token.
+
+. The final prerequisite before your first cluster deployment is making sure the necessary account-wide roles and policies are created. The `rosa` CLI can help with that by using the command shown under _2.2. To create the necessary account-wide roles and policies quickly…_. The alternative to that is manual creation of these roles and policies.
```

Comment:
What is the 2.2 mean?If it is in the doc.It would be better to add a link for it